### DISCUSSION TITLE
CTRL + Enter executes SQL and opens context menu in SQL Editor

### DISCUSSION THREAD

**Discussion by joaorafael-revinate** [dbeaver/dbeaver#35727] [Q&A] [answered]
Hello everyone,

In the latest version of DBeaver, there seems to be an issue with the keyboard shortcut CTRL + Enter in the SQL Editor. In previous versions, this combination executed the SQL instruction as expected. However, after updating, it now both executes the SQL instruction and simultaneously opens a context menu, which is disruptive and undesirable. This behavior was not present in previous versions of DBeaver. It seems to be a conflict introduced with a recent update.

I have tried looking into the settings and keybindings to disable or change the context menu behavior but have not found a solution. 

![image](https://github.com/user-attachments/assets/e44a2ffa-16b0-49e0-9075-45efaa19ee73)

**Environment**
DBeaver Version: Version 24.2.1.202409221725
OS: macOS Sequoia Version 15.0 (this was also verified in previous macOS version)

Does anyone know how to prevent it?

**Comment by TB-HunBoka** [comment, marked as answer]
Duplicate, check the comments in #35109

**Reply by joaorafael-revinate** [reply]
Thank you so much @TB-HunBoka. I had searched before opening this ticket but was unable to find it.
